views/js/runner/proxy.js

```diff
@@ -472,32 +472,34 @@ define([
              * @param {Object} variables
              * @returns {Promise} - Returns a promise. The result of the request will be provided on resolve.
              *                      Any error will be provided if rejected.
+             * @param {Boolean} deferred whether action can be scheduled (put into queue) to be sent in a bunch of actions later.
```

Comment:
please move `@param` definition before `@retrurns`

Reply:
I assume the default value is `false` ? Is this is the case, please document it.